src/Test.UnitTests.Sarif.WorkItems/SarifWorkItemExtensionsTests.cs

```diff
@@ -52,26 +53,109 @@ public void SarifWorkItemExtensions_CreateWorkItemTitle_HandlesSingleResultWithR
             sb.Length.Should().Be(0, because: Environment.NewLine + sb.ToString());
         }
 
+        private class PhraseToolNamesTestCase
+        {
+            public PhraseToolNamesTestCase(List<string> input, string expectedOutput)
+            {
+                Input = input;
+                ExpectedOutput = expectedOutput;
+            }
+
+            public List<string> Input { get; }
+            public string ExpectedOutput { get; }
+        }
+
+        private static readonly ReadOnlyCollection<PhraseToolNamesTestCase> s_phraseToolNamesTestCases =
+            new ReadOnlyCollection<PhraseToolNamesTestCase>(new PhraseToolNamesTestCase[]
+            {
+                new PhraseToolNamesTestCase(
+                    input: new List<string>() {""},
+                    expectedOutput: ""),
```

Comment:
>"" [](start = 36, length = 2)

I suggest your helper function emits `''` here instead of nothing, to make it clearer to the reader that something is missing.

Reply:
This feels a little odd to me, in that I wouldn't expect anything returned for a blank item, but I don't have a strong objection.

---
In reply to: [398963914](https://github.com/microsoft/sarif-sdk/pull/1831#discussion_r398963914) [](ancestors = 398963914)